-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-3eadf5/35429/summary.html
**COMMIT**: 73b60b663f322eed8bb375782e3f1ee390a30329
**CMSSW**: CMSSW_13_3_DEVEL_X_2023-10-25-2300/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/43118/35429/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/DQMServices/Core/src/MonitorElement.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/DQMServices/Core/src/QReport.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/DQMServices/Core/src/QTest.cc
>> Compiling  /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/DQMServices/Core/src/ROOTFilePB.pb.cc
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/DQMServices/Core/src/DQMNet.cc: In member function 'void DQMBasicNet::reserveLocalSpace(uint32_t)':
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/DQMServices/Core/src/DQMNet.cc:1216:67: error: 'DQMImplNet<DQMNet::Object>::ObjectMap' {aka 'class std::unordered_set<DQMNet::Object, DQMNet::HashOp, DQMNet::HashEqual, std::allocator<DQMNet::Object> >'} has no member named 'resize'; did you mean 'size'?
 1216 | void DQMBasicNet::reserveLocalSpace(uint32_t size) { local_->objs.resize(size); }
      |                                                                   ^~~~~~
      |                                                                   size
In file included from /cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02808/el8_amd64_gcc12/cms/cmssw/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/DataFormats/Common/interface/OrphanHandle.h:22,
                 from /cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02808/el8_amd64_gcc12/cms/cmssw/CMSSW_13_3_DEVEL_X_2023-10-25-2300/src/FWCore/Framework/interface/Event.h:23,
</pre>

--- a/DQMServices/Core/src/DQMNet.cc
+++ b/DQMServices/Core/src/DQMNet.cc
@@ -1213,7 +1213,7 @@ DQMBasicNet::DQMBasicNet(const std::string &appname /* = "" */) : DQMImplNet<DQM
 }
 
 /// Give a hint of how much capacity to allocate for local objects.
-void DQMBasicNet::reserveLocalSpace(uint32_t size) { local_->objs.resize(size); }
+void DQMBasicNet::reserveLocalSpace(uint32_t size) { local_->objs.reserve(size); }
 
 /// Update the network cache for an object.  The caller must call
 /// sendLocalChanges() later to push out the changes.

